lib/fathead/mdn_css/redirect_urls.yml

```diff
@@ -0,0 +1,112 @@
+---
+redirects:
+  -moz-appearance: -webkit-appearance
+  :any: :-moz-any
+  :any-link: :-moz-any-link
+  :fullscreen: :-moz-full-screen
```

Comment:
Do we also have redirects for the normalized title? i.e. `moz full screen` -> `:fullscreen`